water/lib/Dialect/Wave/Transforms/LowerReadWriteOps.cpp

```diff
@@ -88,6 +88,12 @@ materializeAffine(Location loc, ArrayRef<wave::WaveSymbolAttr> symbols,
       v = blockId(gpu::Dimension::y);
     else if (name == "_WG2")
       v = blockId(gpu::Dimension::z);
+    else if (name == "_DD0")
+      v = blockId(gpu::Dimension::x);
+    else if (name == "_DD1")
+      v = blockId(gpu::Dimension::y);
+    else if (name == "_DD2")
+      v = blockId(gpu::Dimension::z);
```

Comment:
Why is this using `blockId`?

Reply:
Current wave implementation is a placeholder implementation with `arith.constant(index, 0)`.
I implemented the same placeholder in water.